### PR TITLE
Move the Administering guide upstream

### PR DESCRIPTION
**Pull request by spetrosi**
Bug 1843476 - [upstream] Move the Administering guide upstream

https://bugzilla.redhat.com/show_bug.cgi?id=1843476

### CODE REVIEW

File: guides/doc-Administering_Red_Hat_Satellite/topics/Configuring_Identity_Management.adoc [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+# ipa-client-install
+----
+. Create a realm proxy user, `realm-{smart-proxy-context}`, and the relevant roles in {FreeIPA}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# foreman-prepare-realm admin realm-{smart-proxy-context}
+----
++
+Note the principal name that returns and your Identity Management server configuration details because you require them for the following procedure.
```
[the remark this answers is not in the document]
Comment (spetrosi) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Configuring_Identity_Management.adoc:161]: From Wiki [1]:
FreeIPA is a free and open source identity management system.
I think it is fine to refer to both FreeIPA and IdM servers as identity management servers.

https://en.wikipedia.org/wiki/FreeIPA

File: guides/doc-Administering_Red_Hat_Satellite/topics/Using_IdM.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[[sect-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Configuring_External_Authentication-Using_Identity_Management]]
+=== Using Identity Management
```
[the remark this answers is not in the document]
Comment (tahliar) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Using_IdM.adoc:2]: Clicked around the docs on the customer portal and it looks like it's Red Hat Identity Management in the RHEL 7 docs, but the RHEL 8 docs seem to drop the Red Hat part and just call it Identity Management.

File: guides/doc-Administering_Red_Hat_Satellite/topics/Configuring_Identity_Management.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (tahliar) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Configuring_Identity_Management.adoc:161]: Might be better to have it in lower case, then, for when it refers to both.

File: guides/doc-Administering_Red_Hat_Satellite/topics/Using_IdM.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (spetrosi) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Using_IdM.adoc:2]: Yup, it is an Identity Management. My fault, I looked at the wrong place on the Internet. I will change the attribute. Identity Management is a wide term, so I think adding Red Hat will help to narrow it to the Red Hat software.

File: guides/doc-Administering_Red_Hat_Satellite/topics/creating-a-user.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+. In the *Description* field, add a description of the new user.
+. Select a specific language for the user from the *Language* list.
+. Select a timezone for the user from the *Timezone* list.
++
+By default, {ProjectServer} uses the language and timezone settings of the user’s browser.
+
+. Set a password for the user:
+.. From the *Authorized by* list, select the source by which the user is authenticated.
+- *INTERNAL*: to enable the user to be managed inside {ProjectServer}.
+- *LDAP or {FreeIPA}*: to configure external authentication as described in xref:chap-{Project_Link}-Administering_{Project_Link}-Configuring_External_Authentication[].
```
[the remark this answers is not in the document]
Comment (spetrosi) [reply guides/doc-Administering_Red_Hat_Satellite/topics/creating-a-user.adoc:24]: Rights, the label in this doc is outdated, it is supposed to be EXTERNAL.

File: guides/doc-Administering_Red_Hat_Satellite/topics/Configuring_Identity_Management.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (spetrosi) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Configuring_Identity_Management.adoc]: Actually it was not referring to the term I mentioned, but to the Red Hat software. I fixed it now with an attribute.

**Comment by tahliar** [comment]
Travis' errors are the usual projects.theforeman.org so that's fine for this PR.

**Pull request merged**